examples/mongodbatlas_federated_settings_identity_provider/azure/README.md

```diff
@@ -0,0 +1,233 @@
+  - `insert_record_fields`
+
+## Resource Table
+
+## Usage
+
+**1\. Ensure your Azure credentials are set up.**
+
+1. Install the Azure CLI by following the steps from the [official Azure documentation](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
+2. Run the command `az login` and this will take you to the default browser and perform the authentication.
+3. Once authenticated, it will print the user details as below:
+
+```shell
+⇒  az login
+You have logged in. Now let us find all the subscriptions to which you have access...
+The following tenants don't contain accessible subscriptions. Use 'az login --allow-no-subscriptions' to have tenant level access.
+XXXXX
+[
+  {
+    "cloudName": "AzureCloud",
+    "homeTenantId": "XXXXX",
+    "id": "XXXXX",
+    "isDefault": true,
+    "managedByTenants": [],
+    "name": "Pay-As-You-Go",
+    "state": "Enabled",
+    "tenantId": "XXXXX",
+    "user": {
+      "name": "person@domain.com",
+      "type": "user"
+    }
+  }
+]
+```
+
+**2\. Ensure your MongoDB Atlas credentials are set up.**
+
+This can be done using environment variables:
+
+```shell
+export MONGODB_ATLAS_PUBLIC_KEY="xxxx"
+export MONGODB_ATLAS_PRIVATE_KEY="xxxx"
+```
+
+**3\. TFVARS**
+
+Create a file named `vars.auto.tfvars`
+
+```hcl
+project_name             = "tf-example-oidc"
+location                 = "eastus" # Azure region
+owner                    = "my-user"
+org_id                   = "YOUR_ATLAS_ORG_ID" # e.g 65def6ce0f722a1507105aa5
+region                   = "US_EAST_1" # Atlas region
+insert_record_database   = "test"
+insert_record_collection = "test"
+ssh_public_key           = "ssh-rsa AAA...." # see below for how you can configure this
+```
+
+Configuring the `ssh_public_key` for the Azure VM:
+
+```shell
+cd ~/.ssh
+ssh-keygen -t rsa -b 4096 -C "<your_email@example.com>" # to generate a keypair for the VM
+export TF_VAR_ssh_public_key=$(cat ~/.ssh/id_rsa.pub) # set the `ssh_public_key` with an env var instead of using the variable
+```
+
+**4\. Import the `mongodbatlas_federated_settings_org_config`**
+
+replace `{federated_settings_id}` and `{org_id}` and run:
+
+```shell
+terraform init
+terraform import mongodbatlas_federated_settings_org_config.this {federated_settings_id}-{org_id}
+```
+
+**5\. Review the Terraform plan.**
+
+Execute the below command and ensure you are happy with the plan.
+
+```shell
+terraform plan
+```
+
+**6\. Execute the Terraform apply.**
+
+Now execute the plan to provision the resources.
+
+```shell
+terraform apply
+```
+
+**7\. Connect to MongoDB to verify the record has been inserted.**
+
+- Get the connection string by running `terraform output -json | jq -r '.user_test_conn_string.value'`
+- Open your preferred tool, e.g., [MongoDB Compass](https://www.mongodb.com/products/tools/compass)
+- You should see a new record inserted in `{database}`, `{collection}`, e.g.: `{"_id": "6661790007beeb09e3f1b914", "hello": "world", "ts": "2024-06-06T08:53:20.125919"}`
```

Comment:
probably good to specify that records are inserted into a collection named `test` inside a database named `test`

Reply:
Added in [4ae3870](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2334/commits/4ae3870bef3b4a04aaf571958a011181cb6949d8)